subprojects/docs/src/docs/release/notes.md

```diff
@@ -79,6 +79,32 @@ The [PMD](userguide/pmd_plugin.html) and [CodeNarc](userguide/pmd_plugin.html) p
 
 In Java projects, these tools will use the same version of Java required by the project. In other types of projects, they will use the same version of Java that is used by the Gradle daemon.
 
+#### Introduced `projectInternalView()` dependency for test suites with access to project internals
+The [JVM test suite](userguide/jvm_test_suite_plugin.html) `dependencies` block now
+supports depending on the internal view of the current project during compile-time.
+Previously it was only possible to depend on the current project's API. This allows
+test suites to access project internals that are not declared on
+the `api` or `compileOnlyApi` configurations. This functionality can be useful when
+testing internal classes that use dependencies which are not exposed as part of a 
+project's API, like those declared on the `implementation` and `compileOnly` configurations.
+This dependency would likely prove useful when defining a custom suite for unit testing. 
```

Comment:
```suggestion 
```
I prefer to avoid equivocating like "would likely prove useful" in the docs, and I suspect the example below demonstrates this idea enough that we don't need this sentence.